src/index.js

```diff
@@ -7,8 +7,14 @@ import parse from './parse.js'
 async function run() {
   core.info('Parsing issue body ...')
 
+  let content = core.getInput('body');
```

Comment:
This should have been formatted by `prettier`, not sure why the commit trigger didn't run? Could you run `prettier --write src`?

Reply:
Done in [e84d7b8](https://github.com/zentered/issue-forms-body-parser/pull/27/commits/e84d7b8a62106e9991959fa0773af55225775494)

Reply:
> not sure why the commit trigger didn't run

Because initially, I didn't run `npm install`. So I hacked before noticing husky was here and was doing things on commit.